src/main/kotlin/com/duda/whosgoing/models/Event.kt

```diff
@@ -0,0 +1,11 @@
+package com.duda.whosgoing.models
+import java.time.LocalDateTime
+
+class Event (
+
+    title: String,
+    address: String,
+    waw: LocalDateTime,
```

Comment:
`waw`? Nao era pra ser `when`?

Reply:
'when' é palavra reservado do sistema.
'waw' eu tirei de 'when and where', mas podemos botar outro nome na variável.

Reply:
Se pa usa `dateTime` entao (ou so `date` mesmo)